code/__DEFINES/jobs.dm

```diff
@@ -40,4 +40,4 @@
 #define CHAPLAIN			(1<<10)
 #define CLOWN				(1<<11)
 #define MIME				(1<<12)
-#define ASSISTANT			(1<<13)
```

Comment:
What happened here

Reply:
Not sure. Never touched that area. Assistant is still there.